src/polymer/js-utils.ts

```diff
@@ -101,10 +101,38 @@ export function toScannedMethod(
   if (scannedMethod.type === 'Function' ||
       scannedMethod.type === 'ArrowFunction') {
     const value = <estree.FunctionExpression>node.value;
-    scannedMethod.params = (value.params || []).map((param) => {
-      // With ES6 we can have a lot of param patterns. Best to leave the
-      // formatting to escodegen.
-      return {name: escodegen.generate(param)};
+
+    const paramTags = {};
+    if (scannedMethod.jsdoc) {
+      for (const tag of (scannedMethod.jsdoc.tags || [])) {
+        if (tag.tag === 'param' && tag.name) {
+          paramTags[tag.name] = tag;
+
+        } else if (tag.tag === 'returns') {
```

Comment:
According to https://github.com/google/closure-compiler/wiki/Types-in-the-Closure-Type-System#typedefs, `@return` is the correct tag, though both work.